forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -171,17 +173,15 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Captures a Forage Payment associated with an EBT card
-     *
-     * @param params The parameters required for payment capture, including
-     * reference to a ForagePINEditText and a Payment ref
+     * Captures a payment via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element.
```

Comment:
To help contrast this method against `deferPaymentCapture`, we should using contrasting phrasing in the descriptions for `capturePayment`

```suggestion
     * Immediately captures a payment via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element client-side.
```

whereas `deferPaymentCapture` collects the customer's PIN to be captured later from the server.